docs/howtos/install_epinio_on_rancher_desktop.md

```diff
@@ -53,5 +53,24 @@ $ helm install epinio -n epinio --create-namespace epinio/epinio --set global.do
 $ epinio settings update
 ```
 
-> NOTE: there is currently a [blocking issue](https://github.com/rancher-sandbox/rancher-desktop/issues/576) on Linux which prevent Epinio to push application!
-> However, you will find a workaround at the end of the issue.
+:::caution
+
+For RancherDesktop on Linux, in order to be able to open the port `443` and access the URL set in `global.domain` (i.e. 127.0.0.1.sslip.io), you need to add the port to the `unprivileged` list:
```

Comment:
Side note, that's not totally true, the command does not add the port to unprivileged list, it lowers the range of unprivileged port from 1000 to 443. So 444, 445, 446 etc will become unprivileged too.